docs/api/python/symbol.md

```diff
@@ -480,6 +480,37 @@ Composite multiple symbols into a new one by an operator.
     Custom
 ```
 
+## Contrib
+
+```eval_rst
+.. warning:: This package contains experimental APIs and may change in the near future.
+```
+
+The `contrib.symbol` module contains many useful experimental APIs for new features. This is a place for the community to try out the new features, so that feature contributors can receive feedback.
+
+```eval_rst
+.. currentmodule:: mxnet.contrib.symbol
+
+.. autosummary::
+    :nosignatures:
+
+    CTCLoss
```

Comment:
I thought symbols and ndarrays have listed all aliases. Anyway I will get rid of one here.